File: javascript/7kyu/Power of two.js
// https://www.codewars.com/kata/534d0a229345375d520006a0

// Complete the function power_of_two/powerOfTwo (or equivalent, depending on your language) that determines if a given non-negative integer is a power of two

function isPowerOfTwo(n){
    if(n == 1)
        return true
    if(n < 1)
        return false
    return isPowerOfTwo(n / 2)
}

File: java/7kyu/Power of two.java
// https://www.codewars.com/kata/534d0a229345375d520006a0

// Complete the function power_of_two/powerOfTwo (or equivalent, depending on your language) that determines if a given non-negative integer is a power of two

public class PowerOfTwo {
	public static boolean isPowerOfTwo(long n) {
		return (n > 0) && ((n & (n - 1)) == 0);
	}
}